data linkage enhancement project page (#77)
nhsengland/datascience: docs/our_work/linkage.md

---
title: 'Data Linkage Enhancement'
summary: 'The data linkage enhancement project encompasses all things data linkage, from documenting the existing state of linkage in NHS England in the MPS handbook, to exploring better matching algorithms using Splink, to creating a quality assurance framework for data linkage.'
origin: 'NHS Digital'
tags: ['BEST PRACTICE', 'LINKAGE', 'PYTHON', 'QUALITY']
---

> ![Diagram representing the three current areas of the data linkage project: Quality Assurance, Better Matching Algorithm, and Data Linkage as a service.](../images/linkage_visual_abstract.png)

Reviewer: Love it

> Data linkage is an essential part of being able to use data within any organisation, but in particular the NHS. Being able to trace patients across their care journey, making underrepresented populations not be lost in the cracks, and ensuring compatibility when using several data sets for the same purpose is a pillar of why the data linkage team exists, and why we do what we do. 

Reviewer: ```suggestion
Data linkage is an essential part of being able to use data within any organisation, but in particular the NHS. Being able to trace patients across their care journey, making sure that underrepresented populations are not lost in the cracks, and ensuring compatibility when using several data sets for the same purpose is a pillar of why the data linkage team exists, and why we do what we do. 
```

> # **Work we do**

Reviewer: make it a smaller category of heading, e.g. ##, and the ones below smaller still, e.g. ###

> # **Work we do**

Reviewer: No mention of the linkage community?

> ## Data Linkage as a service

Reviewer: Order these as they are in the diagram - for me, the Service comes last, as you've got better algorithms, you've set up a quality assurance framework - but you're not just leaving people to figure it out themselves - you're also setting up a service to actually help them.

> This is the umbrella to everything else we do. We want to make linkage as easy as possible for teams within the NHS, but also help external researchers find the information about their patient cohorts that they require, and whilst the NHS number is a starting point for this, it isn't perfect and not all data sets even have NHS numbers for all patients! 

Reviewer: ```suggestion
This is the umbrella covering everything else we do. We want to make linkage as easy as possible for teams within the NHS, but also help external researchers find the information about their patient cohorts that they require, and whilst the NHS number is a starting point for this, it isn't perfect and not all data sets even have NHS numbers for all patients! 
```

Reviewer: Any more information on what the service actually entails - it's plans? Any acheivements?

## Better Matching Algorithm

> We have worked on [documenting MPS](https://digital.nhs.uk/services/personal-demographics-service/master-person-service/the-person_id-handbook), to make the current process of linking data in the NHS more transparent and easy to understand, so people know the data they're getting using it. But at the same time, we're constantly working on making it better, currently working on implementing a [probabilistic linkage model](https://www.bristol.ac.uk/media-library/sites/cmm/migrated/documents/problinkage.pdf), in order to improve linkage outcomes, and by extension, patient outcomes. 

Reviewer: ```suggestion
We have worked on [documenting MPS](https://digital.nhs.uk/services/personal-demographics-service/master-person-service/the-person_id-handbook), to make the current process of linking data in the NHS more transparent and easy to understand, giving colleagues to better understanding of the data they get back from MPS. We're also working on making MPS better, currently focussing on implementing a [probabilistic linkage model](https://www.bristol.ac.uk/media-library/sites/cmm/migrated/documents/problinkage.pdf), in order to improve linkage outcomes, and by extension, patient outcomes. 
```

Reviewer: I just reworded it a bit, to hopefully make it flow slightly better.

## Quality Assurance Framework

> A big part of what we do is how do we do linkage, but the underlying question is, how do we do linkage **well**. Creating a quality assurance framework for anyone within the NHS or externally to use who is wanting to link data is a big part of our goals. 

Reviewer: ```suggestion
In our work, it's crucial we ask the question: "How do we do linkage **well**?". Creating a quality assurance framework for anyone within the NHS or externally to use who is wanting to link data is a big part of our goals. 
```

> 

Reviewer: Any scope for a links section as seen in other pages? It would be good if we can follow a standard format. you could link to the githubs, MPS handbook, etc.
